css/style.css

```diff
@@ -16,4 +18,293 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+p {
 
+margin-top: 1 rem;
+margin-bottom:0;
+
+
+
+}
+
+ .header {
+
+height: 5rem;
```

Comment:
It's helpful to keep the indentation consistent when formatting CSS. 
Having a consistent one tab indentation inside the `{}`makes it much easier to see what style attributes are inside what definitions.